deployment/pcm/README.md

```diff
@@ -0,0 +1,429 @@
+- VerticalPodAutoscaler (value: `verticalPodAutoscaler.enabled`, default: false)
+
+Here are available methods in this chart of metrics collection w.r.t interfaces and required access:
+
+| Method                  | Used interfaces      | default | Notes                                                                                                   | instructions                                                               |
+|-------------------------|----------------------| ------- | ------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------- |
+| unprivileged "indirect" | perf, resctrl        |    v    | recommended, missing metrics: energy metrics  (TODO link to issues/PR or node_exporter/rapl_collector)  | `helm install . pcm`                                                       |
+| privileged "indirect"   | perf, resctrl        |         | not recommended, unsecure, no advantages over unprivileged), missing metrics: energy metrics            | `helm install . pcm --set privileged=true`                                 |
+| privileged "direct"     | msr                  |         | not recommended, unsecure and requires msr module pre loaded on host                                    | `helm install . pcm -f values-direct-privileged.yaml`                      |
+| unprivileged "direct"   | msr                  |         | not recommended, requires msr module and access to /dev/cpu  and /dev/mem (non trivial, like using 3rd plugins) | [link for detailed documentation](docs/direct-unprivileged-deployment.md)  |
+
+For more information about direct/indirect collection methods please see [here](#metric-collection-methods-capabilites-vs-requirements)
+
+#### Integration features:
+
+- node-feature-discovery based nodeSelector and nodeAffinity (values: `nfd`, `nfdBaremetalAffinity`, `nfdRDTAffinity`),
+- Examples for non-privileged mode using device plugin ("smarter-devices-manager") or using NRI device-injector plugin (TODO) (file: [values-smarter-devices-cpu-mem.yaml](values-smarter-devices-cpu-mem.yaml) ),
+- Integration with NRI balloons policy plugin (value: `nriBalloonsPolicyIntegration`),
+
+#### Debugging features:
+
+- Local image registry for development (file: [values-local-image.yaml](values-local-image.yaml) ),
+- Deploy Prometheus operator' PodMonitor (value: `podMonitor`)
+
+### Getting started
+
+#### Indirect non-privileged method using Linux abstractions (perf/resctrl) default.
+
+```sh
+helm install pcm . 
+```
+
+#### Direct privileged method
+```
+helm install pcm . -f values-direct-privileged.yaml
+```
+
+#### All opt-in features: Node-feature-discovery + Prometheus podMonitor + vertical
+
+```
+helm install ... --set nfd=true --set podMonitor=true --set verticalPodAutoscaler.enabled=true
+```
+
+### Requirements
```

Comment:
I think you could add information on how to check it or enable all requirements.